.github/ISSUE_TEMPLATE/propose-a-talk.md

```diff
@@ -18,6 +18,11 @@ assignees: ''
 #### Description
 <!-- Please include brief abstract about your talk. -->
 
+#### Table of contents
```

Comment:
@bhavin192 I think abstract of the talk is enough to know about the talk proposal. Is table of contents really required at the time of proposal? I believe keeping proposal submission as less as possible make it easy for the submitter, isn't it?

I am not expert but definitely would like to hear opinion from all.

Reply:
I get what you are trying to say here. From the past experience, it's not at all clear most of the time that what is going to be covered in a talk with just the abstract. Some people just put one or two liner summary. Having a small ToC definitely helps to understand what will be covered in the talk. (We have received a feedback from one attendee that, we don't have clear detailo about the talks)